be/src/runtime/client_cache.h

```diff
@@ -148,7 +148,7 @@ class ClientConnection {
 public:
     ClientConnection(ClientCache<T>* client_cache, TNetworkAddress address, Status* status)
```

Comment:
Better not change the timeout here, if user want to specify the timeout param, use the other constructor.